litellm/proxy/db/exception_handler.py

```diff
@@ -43,6 +43,35 @@ def is_database_connection_error(e: Exception) -> bool:
         if isinstance(e, ProxyException) and e.type == ProxyErrorTypes.no_db_connection:
             return True
         return False
+    
+    @staticmethod
+    def is_database_retriable_exception(e: Exception) -> bool:
+        """
+        Returns True if the execption is from a condition (e.g. deadlock, broken connection, etc.) that should be retried.
+        """
+        import re
+
+        if isinstance(e, DB_CONNECTION_ERROR_TYPES): # TODO: is this actually needed?
+            return True
```

Comment:
I am maintaining this just because it was the old logic. I have no idea, nor a way to practically test, if this is appropriate or not.